sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/ScanBuilder.java

```diff
@@ -23,7 +23,8 @@
  * An interface for building the {@link Scan}. Implementations can mixin SupportsPushDownXYZ
  * interfaces to do operator push down, and keep the operator push down result in the returned
  * {@link Scan}. When pushing down operators, the push down order is:
- * sample -&gt; filter -&gt; aggregate -&gt; limit -&gt; column pruning.
+ * sample -&gt; filter -&gt; aggregate -&gt; offset -&gt; limit/top-n(sort + limit) -&gt;
```

Comment:
We can switch the order of LIMIT and OFFSET to match this defined order, but we can't switch the order of ORDER BY and OFFSET. What if the SQL query is `ORDER BY ... LIMIT ... OFFSET ...`? Nothing can be pushed down?